multi-node/aws/pkg/config/config_test.go

```diff
@@ -5,7 +5,7 @@ import (
 	"testing"
 )
 
-const minimalConfigYaml = `externalDNSName: test-external-dns-name
+const minimalConfigYaml = `externalDNSName: test.example.com
```

Comment:
should we make this `staging.core-os.net` to be consistent with `cluster_test.go`?